ops/terraform/modules/resources/templates/pipeline_server.tpl

```diff
@@ -13,7 +13,9 @@ git clone https://github.com/CMSgov/beneficiary-fhir-data.git --branch ${gitBran
 
 cd beneficiary-fhir-data/ops/ansible/playbooks-ccs/
 
-git checkout ${gitCommitId}
+# At this time gitCommitId is a unique merge commit in Jenkins that cannot be properly
+# checked out via GitHub, uncommenting this will break instance launch!
+# git checkout ${gitCommitId}
```

Comment:
Disabling this feature here until it's refactored to work as intended.

Reply:
Is there a ticket to make something that work.

Reply:
No, I'll go ahead and make one.